src/components/ConnectedFilter/index.jsx

```diff
@@ -153,7 +166,7 @@ class ConnectedFilter extends React.Component {
         tabs={filterTabs}
         filterConfig={this.props.filterConfig}
         onFilterChange={e => this.handleFilterChange(e)}
-        hideZero={this.props.hideZero}
+        hideZero={this.props.hideZeroCountFilterOptions}
```

Comment:
Is the reason why we change the name `hideZero` to `hideZeroCountFilterOptions` because of backward compatibility here?